### DISCUSSION TITLE
Not Connected to Lightroom Message

### DISCUSSION THREAD

**Discussion by TerryHewitt** [rsjaffe/MIDI2LR#630] [Problems]
Not sure what I've done but MID2LR suddenly displaying the above message (and not working of course).
Running Windows 10. MIDI2LR 5.3.0.2, Lightroom 11.2
I think I've opened ports 58763 and 58764 in my firewall. TCP or UDP?
I've disabled and re-enabled the plugin. I've reinstalled it. I've tried 5.3.0.0 as well.

These appear to be the critical two lines in midi2lr.log

2022-02-13T17:59:24.042Z LR_IPC_Out.cpp(134) LR_IPC_Out did not connect. No connection could be made because the target machine actively refused it..
2022-02-13T17:59:24.043Z LR_IPC_In.cpp(121) LR_IPC_In did not connect. No connection could be made because the target machine actively refused it..

Can anyone advise how to fix please?

Best wishes 
Terry

**Comment by TerryHewitt** [comment]
Its working now!

Still don't know what I've done right/wrong...

Sorry to bother you all

Best wishes

Terry

**Reply by edrudolph** [reply]
New user here, trying to use an Arturia Beatstep with MIDI2LR 5.4.0.  Mac OS 11.7.2.  I'm getting the Not Connected to Lightroom error.  Have restarted but am unsure what else to do
